Source/Web/app/information.js

```diff
@@ -653,7 +729,13 @@ function dispWork(id) {
                     "" +
                     "<div class='form-group'><label class='control-label col-sm-2' for='text'>Worker Surname:</label>" +
                     "<div class='col-sm-9'><p class='form-control-static'>" + snapshot.val().surname + "</p> </div> </div>" +
-                    "" +
+		    "" +
+                    "<div class='form-group'><label class='control-label col-sm-2' for='text'>Identity Number:</label>" +
+                    "<div class='col-sm-9'><p class='form-control-static'>" + snapshot.val().idNumber + "</p> </div> </div>" +
+		    "" +
+		    "<div class='form-group'><label class='control-label col-sm-2' for='text'>Phone Number:</label>" +
+                    "<div class='col-sm-9'><p class='form-control-static'>" + snapshot.val().phoneNumber + "</p> </div> </div>" +
+		    "" +
                     "<div class='form-group'><label class='control-label col-sm-2' for='sel1'>Assigned Orchard:</label>" +
                     "<div class='col-sm-9'><span id='workOrchDisp'></span></div></div>" +
                     "" +
```

Comment:
When the foreman option is selected it must ask for the phone number and not the email anymore.